modules/rke2/worker/instances_worker.tf

```diff
@@ -47,11 +94,53 @@ locals {
   master_ip = trimspace("${data.local_file.master_ip.content}")
 }
 
+data "local_file" "master_ip_prev" {
+  depends_on = [var.dependency]
+  filename = "/tmp/${var.resource_name}_master_ip_prev"
+}
+
+locals {
+  master_ip_prev = trimspace("${data.local_file.master_ip_prev.content}")
+}
+
 data "local_file" "token" {
   depends_on = [var.dependency]
   filename = "/tmp/${var.resource_name}_nodetoken"
 }
 
 locals {
   node_token = trimspace("${data.local_file.token.content}")
+}
+
+resource "null_resource" "stop_all_nodes" {
+  count = var.create_eip ? 1 : 0
+  depends_on = [null_resource.worker_eip]
+   provisioner "local-exec" {
+   command = "chmod +x ../install/rke2_stop_start_instance.sh"
+  }
+  provisioner "local-exec" {
+    command = "../install/rke2_stop_start_instance.sh stop ${var.resource_name}"
+  }
+}
+
+resource "time_sleep" "wait_for_stop" {
+  count = var.create_eip ? 1 : 0
+  create_duration = "400s"
+  depends_on = [null_resource.stop_all_nodes]
+}
+
+resource "null_resource" "start_server1_server2" {
+  count = var.create_eip ? 1 : 0
+  depends_on = [time_sleep.wait_for_stop]
+  provisioner "local-exec" {
+    command = "../install/rke2_stop_start_instance.sh start_s1_s2 ${var.resource_name}"
+  }
+}
+
+resource "null_resource" "start_master_worker" {
+  count = var.create_eip ? 1 : 0
+  depends_on = [null_resource.start_server1_server2]
+  provisioner "local-exec" {
+    command = "../install/rke2_stop_start_instance.sh start_master_worker ${var.resource_name}"
+  }
```

Comment:
I believe this is what you're changing to use the aws sdk, but just in case it's not, I think we should create a testcase specifically that does this via the aws sdk. That way we can do the steps like:
1. Bring up the instances with the EIP
2. Deploy a few workloads and validate they are functional
3. Reboot the instances via stop and start like this (but via the aws sdk)
4. Confirm nodes and workloads still functional
5. Deploy a few more workloads and validate they are functional